src/cli/env.js

```diff
@@ -1,3 +1,10 @@
 export const parseEnv = () => {
-    // Write your code here 
-};
\ No newline at end of file
+    process.env['RSS_KEY'] = '123';
+    process.env['other_KEY'] = 'NO-RSS-key';
+    for (let arg in process.env) {
+        if (/rss|RSS/.test(arg)) {
```

Comment:
```suggestion
        if (/^rss|^RSS/.test(arg)) {
```
Если использовать регулярки, то тут нужно добавить спецсимвол начала строки, чтобы работало правильно. Иначе посчиают валидными и строки, где `'RSS'` где-нибудь посередине, а нужно было, чтобы имя начиналось с `'RSS'`